packages/tailwind-config/shared-styles.css

```
@@ -43,6 +43,7 @@
   --color-brand-text-700: var(--color-slate-700);
   --color-brand-text-800: var(--color-slate-800);
   --color-brand-text-800: var(--color-slate-800);
```

Stylelint (17.7.0)

[error] Duplicate custom property "--color-brand-text-800" (declaration-block-no-duplicate-custom-properties)

(declaration-block-no-duplicate-custom-properties)